### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/react_movies-list/)

### CODE REVIEW

**Review by SerhiiKirik — APPROVED** [review]
Well done 🔥

File: src/App.tsx [hunk 1]
```diff
@@ -1,176 +1,15 @@
...
-              countries, where they each meet a local guy and fall in love.
-              <br />
-              <a href="https://www.imdb.com/title/tt0457939">IMDB</a>
-            </div>
-          </div>
-        </div>
-      </div>
+      <MoviesList
+        movies={moviesFromServer}
+      />
```
Comment (illiarerun) [inline src/App.tsx:12]:
```suggestion
      <MoviesList movies={moviesFromServer}/>
```

File: src/components/MoviesList/MoviesList.tsx [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
```
Comment (illiarerun) [inline src/components/MoviesList/MoviesList.tsx:10]: Move that interface somewhere else and reuse it with Omit in MovieCard component

**Review by illiarerun — CHANGES_REQUESTED** [review]
Good job 🔥 Let's just fix one little thing:

**Review by illiarerun — APPROVED** [review]
GJ 🔥

**Pull request closed without merging**